sdk/ai/azure-ai-projects/README.md

```diff
@@ -695,6 +695,23 @@ message = project_client.agents.create_message(
 
 <!-- END SNIPPET -->
 
+Azure blob storage can be used as a message attachment. In this case `VectorStoreDataSource` have to be used as a data source:
+
+<!-- SNIPPET:sample_agents_code_interpreter_attachment_enterprise_search.create_agent -->
+
+```python
+code_interpreter = CodeInterpreterTool()
+
+# notice that CodeInterpreter must be enabled in the agent creation, otherwise the agent will not be able to see the file attachment
+agent = project_client.agents.create_agent(
+    model="gpt-4-1106-preview",
+    name="my-assistant",
+    instructions="You are helpful assistant",
+    tools=code_interpreter.definitions,
+)
+```
+
```

Comment:
From the code, I am seeing this is the basic code interpreter tool.   This doesn't match to the description you put.

To match the description, you need to have
File upload to blob storage.
Using MessageAttachment class with code interpreter or file search tools
pass the MessageAttachment to the create_message function.

Reply:
Fixing the snippet, I have put the wrong one in the beginning.